project.json

```diff
@@ -18,8 +18,24 @@
     ],
     "restricted": "RobotVacuumSample",
     "gem_names": [
-        "ROS2",
-        "ArchVis"
+        "ArchVis",
+        "Atom",
+        "CameraFramework",
+        "DebugDraw",
+        "GameState",
+        "ImGui",
```

Comment:
I believe that :
ImGui, LandscapeCanvas, SaveData, ScriptCanvasPhysics, WhiteBox are not needed.